proposals/first-class-span-types.md

```diff
@@ -279,6 +282,10 @@ D4 d4 = M4; // Convert void(object[]) to void(string[])
 These conversions may not be possible to do without creating a wrapper lambda without runtime changes; the existing variant delegate conversions are possible to emit
 without needing to create wrappers. We don't have precedent in the language for silent wrappers like this, and generally require users to create such wrapper lambdas themselves.
 
+#### Answer
+
+We will not allow variance in delegate conversions here. `D1 d1 = M1;` and `D2 d2 = M2;` will not compile.
```

Comment:
My take away for this was roughly that overall the thunks seemed like a fine solution, there was just not a belief it was a big enough scenario to warrant the necessary work. Believe that context is important for the answer here. Essentially this is something that is valid to reconsider should the scenarios develop that warrant it.

Reply:
@jaredpar addressed. Any other feedback?